test/kubemark/iks/util.sh

```diff
@@ -106,15 +109,16 @@ function set-registry-secrets {
 function set-hollow-master {
 	echo -e "${color_yellow}CONFIGURING MASTER${color_norm}"
 	master-config
-	MASTER_IP=$(cat $KUBECONFIG | grep server | awk -F "/" '{print $3}')
+	# shellcheck disable=2034 # MASTER_IP used in parent script
+	MASTER_IP=$(grep server "$KUBECONFIG" | awk -F "/" '{print $3}')
 }
 
 # Set up master cluster environment
 function master-config {
 	if [[ "${USE_EXISTING}" = "y" ]]; then
 		export KUBECONFIG=${CUSTOM_MASTER_CONFIG}
 	else
-		$(bx cs cluster-config kubeMasterTester --admin | grep export)
+		eval "$(bx cs cluster-config kubeMasterTester --admin | grep export)"
```

Comment:
what was the lint here? :O

Reply:
[SC2091](https://github.com/koalaman/shellcheck/wiki/SC2091) - _Remove surrounding $() to avoid executing output._

I believe the intent here is to execute it. `bx cs cluster-config kubeMasterTester --admin` outputs something akin to:
 ```
export KUBECONFIG=/Users/<user_name>/.bluemix/plugins/container-service/clusters/pr_firm_cluster/kube-config-prod-par02-pr_firm_cluster.yml`
```
See step 3 here: https://developer.ibm.com/recipes/tutorials/deploy-a-kubernetes-cluster-in-ibm-cloud-container-service#Setting-up-the-Cluster-Environment.

It's sort of like `eval $(minikube docker-env)`

Can swap from eval to storing it as a string and executing it that way if it'd be preferred.

Reply:
ah, this makes sense as is. thanks!